floodviz/static/js/map.js

```diff
@@ -230,6 +232,7 @@
 						}
 					});
 					self.linked_interactions.click(selected);
+					ga_send_event('Map', 'drag_select', selected);
```

Comment:
The variable selected is an array and the function expects a string. You can easily convert to a string by doing selected.join(',')